Лабораторная_4/task_3.py

```diff
@@ -0,0 +1,13 @@
+def delete(list_, index=None):
+      # TODO реализовать функцию удаления элемента из списка по индексу
+      if index == None:
+            list_ = list_[index:-1]
+      elif index == -1:
+           list_ = list_[0:-1]
+      else:
+            list_ = list_[:index] + list_[index + 1:]
+      return list_
```

Comment:
def delete(list_, index=None):
····if index is None:
········index = -1
····if index < 0:
········index = len(list_) + index
····return list_[:index] + list_[index + 1:]